chore: easy wording (#4449)
sveltejs/svelte: site/content/tutorial/01-introduction/03-dynamic-attributes/text.md

## Shorthand attributes

> It's common to have an attribute where the name and value are the same, like `src={src}`. Svelte gives us a convenient shorthand for these cases:

Reviewer: ```suggestion
In case the attribute name and value are the same (like `src={src}`) Svelte provides a convenient shorthand:
```
